### PR TITLE
Release 0.11.22

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Change log
 
+## 0.11.22
+
+### Added
+
+- Kafka extension update dto 1.8.2
```
Comment (GrahamCampbell) [inline CHANGELOG.md:7]: typo ;)

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

**Comment by mnapoli** [comment]
Build fixed in the `prepare-release` branch, and I'm sorry I ended up updating the release to include #311 (since 0.11.22 was never built or release it seems like it shouldn't have bad side effects).

**Comment by Nyholm** [comment]
Great. Thank you for fixing this build. 

I just squash and merge =)

File: CHANGELOG.md (hunk 1 above)
Comment (Nyholm) [reply CHANGELOG.md]: Thank you

**Comment by mnapoli** [comment]
Thank you!

**Pull request merged**